### ISSUE TITLE
Aria-required property is missing for required field in Design screen.

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#870]
**Describe the bug**
Aria-required property is missing for required field.

**To Reproduce**

1. Go to Design  (http://localhost:9992/cm/app/?view=design&pathType=unknown)
2. Click on create new page button. It will open a pop up.
3. "File name" is a required field. Inspect the "File name" text box element, you will find that tha Aria-required=true property is missing for the input text.

**Expected behavior**
Aria-required=true property must be available for all required fields.

**Additional context**
Navigate all the options available for similar issue and fix.
